gendiff/formats/plain.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict
+
+from gendiff.diff import get_diff_view
+
+
+def convert_value_plain_format(val: Any) -> Any:
```

Comment:
this function returns string

Reply:
now it returns Any.
if declire returning str, mypy fails with:

run mypy.................................................................Failed
- hook id: mypy
- exit code: 1
gendiff/formats/plain.py:18: error: Returning Any from function declared to return "str"  [no-any-return]

Reply:
try to declare `exceptions_to_convert: Dict[bool | None, str]`